code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,25 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(is_species(M, /datum/species/squid))
+		if(M.incapacitated())
+			return
+		var/obj/item/I = M.get_active_held_item()
+		var/list/turf/targets = list()
+		var/turf/T = get_turf(M)
+		for(var/turf/T in oview(M, 3))
+		targets += T
+		if(LAZYLEN(targets) && I)
+			to_chat(M, "<span class='warning'>The salt causes your arm to spasm!</span>")
+			owner.log_message("threw [I] due to a Muscle Spasm", LOG_ATTACK)
+			owner.throw_item(pick(targets))
+
+/datum/reagent/consumable/sodiumchloride/reaction_mob(mob/living/M, method=INGEST, reac_volume)
```

Comment:
Literally what?

Reply:
https://youtu.be/dxQmOR_QLfQ

Reply:
I was referring to the fact that you duped the proc instead of just doing an `if(method == INGEST)` check.

Reply:
Thought that one of the parameters being method == touch meant this entire block was method==touch, and I would have to create a separate block for ingest. If I could just put method==ingest in it, easy